lock-api/src/main/java/com/palantir/lock/client/LeasedLockToken.java

```diff
@@ -39,7 +39,7 @@ public final class LeasedLockToken implements LockToken {
     @GuardedBy("this")
     private boolean invalidated = false;
 
-    static LeasedLockToken of(ConjureLockToken serverToken, Lease lease) {
+    public static LeasedLockToken of(ConjureLockToken serverToken, Lease lease) {
         return new LeasedLockToken(serverToken, UniqueIds.pseudoRandomUuidV4(), lease);
```

Comment:
Wait, SOMETHING feels wrong here? Why is this randomly assigning a UUId?

Reply:
You already had that requestid?

Reply:
I left a comment on this. I think this specific id is not used for any purposes. I do a more invasive search